src/browser/modules/Sidebar/GuidesDrawer.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { useRef } from 'react'
+import { connect } from 'react-redux'
+
+import {
+  getGuide,
+  startGuide,
+  Guide,
+  defaultGuide,
+  gotoSlide
+} from 'shared/modules/guides/guidesDuck'
+import { GlobalState } from 'shared/globalState'
+import GuideCarousel from '../GuideCarousel/GuideCarousel'
+import { BackIcon } from '../../components/icons/Icons'
+import {
+  StyledGuidesDrawer,
+  GuideTitle,
+  BackIconContainer,
+  CarouselWrapper,
+  StyledGuidesDrawerHeader
+} from './styled'
+
+type GuidesDrawerProps = {
+  guide: Guide
+  backToAllGuides: () => void
+  gotoSlide: (slideIndex: number) => void
+}
+
+function GuidesDrawer({
+  guide,
+  backToAllGuides,
+  gotoSlide
+}: GuidesDrawerProps): JSX.Element {
+  const scrollRef = useRef<HTMLDivElement>(null)
+  const title = guide.title.startsWith('http')
+    ? guide.title
+    : guide.title.charAt(0).toUpperCase() + guide.title.slice(1)
+
+  return (
+    <StyledGuidesDrawer id="guide-drawer" ref={scrollRef}>
+      <StyledGuidesDrawerHeader onClick={backToAllGuides}>
+        {guide.title !== defaultGuide.title && (
```

Comment:
I find this logic a little hard to follow, makes me wonder if making it some appropriately named utility function could make it more clear. Ex, either `isDefaultGuide(guide)` or some other name that better describes the logic/reasoning for this condition.

Reply:
yes that's better